pages.it/linux/uname.md

```diff
@@ -0,0 +1,36 @@
+# uname
+
+> Uname stampa informazioni a proposito della macchina e del sistema operativo in esecuzione su di essa
+> Maggiori informazioni: <https://www.gnu.org/software/coreutils/manual/html_node/uname-invocation.html>.
+
+- Stampa tutte le informazioni:
+
+`uname --all`
+
+- Stampa il nome corrente del kernel:
```

Comment:
```suggestion
- Stampa il nome del kernel corrente:
```
The way it's worded seems to imply the running kernel can change name. Moving `corrente` at the end is better.